src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1545,6 +1545,8 @@ trait Contexts { self: Analyzer =>
        *     as well as bindings supplied by the compiler but not explicitly written in source code,
        *     have the lowest precedence.
        */
+
+      /** Level 4 (see above) */
```

Comment:
Probably this will induce an additional warning from Scaladoc. "Unmoored" something.